src/DataFactory/DataFactoryV2/DataFactories/UpdateAzureDataFactoryCommand.cs

```diff
@@ -35,6 +37,36 @@ public class UpdateAzureDataFactoryCommand : DataFactoryBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
 
+        #region Attributes
+        [Parameter(Mandatory = false, HelpMessage = Constants.HelpIdentityType)]
+        #endregion
+        public string IdentityType { get; set; }
+
+        #region Attributes
```

Comment:
The change for Update-AzDataFatory doesn't mentioned in design doc. Please add it.

Reply:
Updated the issue: 
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/956